std-bits/table/src/main/java/org/enso/table/data/table/join/JoinKind.java

```diff
@@ -0,0 +1,21 @@
+package org.enso.table.data.table.join;
+
+public enum JoinKind {
```

Comment:
We could include CROSS_JOIN here and merge the builder settings into the Java Enum?

Reply:
I'm not so convinced about this - cross join is using a completely separate implementation (as it has no conditions), I'd rather not couple the 2 together.

They look slightly similar on the outer API side, but on the inside they have more differences than similarities IMHO.